src/main/java/com/dougnoel/sentinel/math/Decimal.java

```diff
@@ -0,0 +1,76 @@
+package com.dougnoel.sentinel.math;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+
+public class Decimal {
+    /**
+     * protected constructor since this is a static class
+     */
+    protected Decimal(){
+    }
```

Comment:
Static classes should always have their constructor declared private for 2 reasons. 1) It prevents the instantiation of multiple versions. 2) It prevents inherited versions from being instantiated.
```suggestion
    private Decimal(){
        //private constructor since this is a static class
    }
```